From 6343555c3a648fa7aa5a41b7fbefa12603b41b0f Mon Sep 17 00:00:00 2001
From: ivan <122584107+wyf9661@users.noreply.github.com>
Date: Sun, 7 Jan 2024 21:15:53 +0800
Subject: [PATCH] Update gcc-wrapper.py to disable interpret_warning

---
 scripts/gcc-wrapper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/gcc-wrapper.py b/scripts/gcc-wrapper.py
index ad9d6a2153ff2..a63d9fd18cb05 100755
--- a/scripts/gcc-wrapper.py
+++ b/scripts/gcc-wrapper.py
@@ -102,7 +102,7 @@ def run_gcc():
         proc = subprocess.Popen(args, stderr=subprocess.PIPE, env=env)
         for line in proc.stderr:
             print (line.decode("utf-8"), end="")
-            interpret_warning(line.decode("utf-8"))
+            #interpret_warning(line.decode("utf-8"))
         if do_exit:
             sys.exit(1)
 
